notebooks/ODE_solver.ipynb

```diff
@@ -11,13 +11,25 @@
     "import json\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        ['N_W', 1/5, {'R':'&gt;1'}], #F_R goes up in dWAPL: Wutz2017 Fig4F 
</pre><p> </p><p>Its probably fine, but it could be confusing that we use the free species symbol for the relative bound ratio of the perturbation .... would be be easier if it was F_R ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Fudenberg-Research-Group/five_state_cohesin/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>